Dockerfile

```diff
@@ -1,38 +1,39 @@
+FROM node:12-alpine as fe-builder
+
+ENV NODE_OPTIONS="--max-old-space-size=4096"
+
+WORKDIR /app/webpack
+COPY webpack ./
+RUN yarn install --frozen-lockfile && \
+    yarn cache clean
+
+COPY zoo/ source/
+RUN yarn production
+
 FROM python:3.7-alpine
 
 ENV DJANGO_SETTINGS_MODULE=zoo.base.settings
 RUN addgroup -S macaque && adduser -H -D -S macaque macaque
 
 WORKDIR /app
+
+COPY --from=fe-builder /app/zoo ./zoo
```

Comment:
I don't understand this line. What is in `/app/zoo`? Because in `fe-builder` I don't see anything being copied there... `zoo` is copied to `/app/webpack/source/` if I understand it correctly.

Reply:
The webpack script generates [that directory](https://github.com/kiwicom/the-zoo/blob/master/webpack/webpack.build.js#L9) 🙂

Reply:
I see :-)